space-concordia-robotics/robotics-prototype
# The Epic Quest to compile C++ with Travis has been completed.

# Assignee Section

## Description
I changed the travis config so it runs cmake and builds it

### Steps for Testing

Given that this PR is only travis config, you cannot test the PR through conventional methods of clicking stuff in our GUI

1. Take a look at this link : https://travis-ci.org/github/space-concordia-robotics/robotics-prototype
2. You will find instances where Travis builds and when it doesnt. 
3. Observe that everything is in order in the latest build for my branch

closes #380 

The approval from all software team leads is necessary before merging.

# Reviewer Section

Aside from local testing and the General Integration Test it is implied that static analysis should be included in the verification process.


For Pull Requests that do not include code changes, it is not required to perform the tests above.
